game/game/fightalgo.cpp

```diff
@@ -298,9 +298,8 @@ float FightAlgo::weaponRange(GameScript &owner, const Npc &npc) {
     case WeaponState::Fist:
       return float(gv.fight_range_fist[gl]);
     case WeaponState::Bow:
-      return float((MaxBowRange*npc.hitChanse(TALENT_BOW))/100);
     case WeaponState::CBow:
-      return float((MaxBowRange*npc.hitChanse(TALENT_CROSSBOW))/100);
+      return float(MaxBowRange);
```

Comment:
Why? Also, this change to AI is not in PR description.

Reply:
Sorry forgot to mention. Before PR changes hit chance was always 100% below this range an 0% above. Now it's variable so it's unclear what this value should be. Maybe `ReferenceRange` to assure a somewhat decent hit chance?

Reply:
Yes, `ReferenceRange` is good choise